processor/src/chiplets/ace/mod.rs

```diff
@@ -0,0 +1,292 @@
+#![allow(dead_code)]
+
+use alloc::{collections::BTreeMap, vec::Vec};
+
+use miden_air::{
+    RowIndex,
+    trace::{chiplets::ace::ACE_CHIPLET_NUM_COLS, main_trace::MainTrace},
+};
+use vm_core::{FieldElement, ZERO, mast::MastNodeExt};
+
+use crate::{
+    ContextId, ExecutionError, Felt, QuadFelt,
+    chiplets::{ace::trace::EvaluationContext, memory::Memory},
+    errors::{AceError, ErrorContext},
+    trace::TraceFragment,
+};
+
+mod trace;
+
+mod encoded_circuit;
+#[cfg(test)]
+mod tests;
```

Comment:
This is what it gives if I format it in a single block 
```suggestion
#![allow(dead_code)]

use crate::{
    ContextId, ExecutionError, Felt, QuadFelt,
    chiplets::{ace::trace::EvaluationContext, memory::Memory},
    errors::{AceError, ErrorContext},
    trace::TraceFragment,
};
use alloc::{collections::BTreeMap, vec::Vec};
use miden_air::{
    RowIndex,
    trace::{chiplets::ace::ACE_CHIPLET_NUM_COLS, main_trace::MainTrace},
};
use vm_core::{FieldElement, ZERO, mast::MastNodeExt};

mod encoded_circuit;
mod trace;

#[cfg(test)]
mod tests;
```

Reply:
I have tried to tidy things a bit, let me know if there is anything we can do better